### PR TITLE
feat : product listing page

### PR DESCRIPTION
**Pull request by dipteekhd**
Added product list page
https://khd-e-commerce.netlify.app/screens/product-list/product-list.html

### CODE REVIEW

File: screens/product-list/product-list.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.filter {
+  display: flex;
+  align-items: center;
+}
+.filter button {
```
Comment (Kevin-Solomon) [inline screens/product-list/product-list.css]: Just a suggestion, be careful using element selectors as every button element in ```filter``` will get these styles.

**Review by Kevin-Solomon — APPROVED** [review]

File: screens/product-list/product-list.html [hunk 2]
```diff
@@ -0,0 +1,381 @@
...
+                            <label for="id-4" class="center-content">
+                                <input id="id-4" type="checkbox" class="m-xs-r">
+                                Diamond
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-4" class="center-content">
+                                <input id="id-4" type="checkbox" class="m-xs-r">
+                                Rings
+                            </label>
```
Comment (RahullRawat) [inline screens/product-list/product-list.html]: - Here you have given same id for multiple labels.
- So whenever i am trying to select gold , diamond or rings it is selecting only second label i.e Earrings.
- You can correct this by giving different id's

File: screens/product-list/product-list.html [hunk 3]
```diff
@@ -0,0 +1,381 @@
...
+                                <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                                2 star & above
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-2" class="center-content">
+                                <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                                1 star & above
+                            </label>
+                        </li>
```
Comment (RahullRawat) [inline screens/product-list/product-list.html]: - Same here you have given same id's

**Review by VishalPatil18 — APPROVED** [review]
The code looks clean. Great work @dipteekhd 👍

**Review by RahullRawat — CHANGES_REQUESTED** [review]

File: screens/product-list/product-list.html [hunk 4]
```diff
@@ -0,0 +1,381 @@
...
+                        <li class="list__input m-xs-all">
+                            <label for="id-1" class="center-content">
+                                <input id="id-1" type="radio" class="m-xs-r" name="I1" checked>
+                                Price - Low to High
+                            </label>
+                        </li>
+                        <li class="list__input m-xs-all">
+                            <label for="id-2" class="center-content">
+                                <input id="id-2" type="radio" class="m-xs-r" name="I1">
+                                Price - High to Low
```
Comment (RahullRawat) [inline screens/product-list/product-list.html]: - Here you have given id as 1 & 2  but you have already given these same id's to ratings
- So suggestion here is to give some different id's here as id's should be unique.

**Review by RahullRawat — CHANGES_REQUESTED** [review]

File: screens/product-list/product-list.html (hunk 2 above)
Comment (dipteekhd) [reply screens/product-list/product-list.html]: Thanks for mentioning will fix

File: screens/product-list/product-list.css (hunk 1 above)
Comment (dipteekhd) [reply screens/product-list/product-list.css]: Sure will fix this!

**Review by RahullRawat — APPROVED** [review]

**Pull request merged**